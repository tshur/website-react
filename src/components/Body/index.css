:local .body {
  background: red;
  height: calc(100% - 100px);
  width: calc(100% - 70px);
  float: right;
  overflow: auto;
}