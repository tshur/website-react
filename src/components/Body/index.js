import React from 'react';
import './index.css';

import Footer from '../Footer';

const Body = (props) =>
  <div>
    <h1> Body </h1>
    <p> {props.match.path} </p>
    <Footer />
  </div>

export default Body;