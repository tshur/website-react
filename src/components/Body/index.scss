.body {
  background: red;
  height: calc(100% - 70px);
  width: 100%;
}