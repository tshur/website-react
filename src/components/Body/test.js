import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Body from '.';

describe('Body', () => {
  const props = {
    match: {
      path: '/'
    }
  }

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Body {...props} />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  test('has a valid snapshot', () => {
    const component = renderer.create(
      <Body {...props} />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
