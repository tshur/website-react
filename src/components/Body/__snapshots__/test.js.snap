// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body has a valid snapshot 1`] = `
<div
  className="container undefined"
>
  <div
    className="container undefined"
  >
    <h1>
       Home 
    </h1>
  </div>
</div>
`;
